discord/client.py

```diff
@@ -1307,7 +1307,7 @@ async def fetch_template(self, code: Union[Template, str]) -> Template:
         data = await self.http.get_template(code)
         return Template(data=data, state=self._connection)  # type: ignore
 
-    async def fetch_guild(self, guild_id: int, /) -> Guild:
+    async def fetch_guild(self, guild_id: int, /, *, with_counts: Optional[bool] = True) -> Guild:
```

Comment:
This type hint is wrong, it should only allow bool

Reply:
it's an Optional Kwarg.
I don't want to break it.

Reply:
`Optional[T]` means `T | None`,  but it only accepts a `bool`

Reply:
do you mean with_counts: bool = True ?

Reply:
Yes

Reply:
Yes, that's correct

Reply:
There you go